presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduClientConfig.java

```diff
@@ -151,4 +155,18 @@ public boolean isGroupedExecutionEnabled()
     {
         return groupedExecutionEnabled;
     }
+
+    @NotNull
+    public Duration getDynamicFilteringProbeBlockingTimeout()
+    {
+        return dynamicFilteringProbeBlockingTimeout;
+    }
+
+    @Config("kudu.dynamic-filtering-probe-blocking-timeout")
+    @ConfigDescription("Duration to wait for completion of dynamic filters during page source generation for probe side table")
```

Comment:
in the spirit of the above, perhaps something like
```
@ConfigDescription("Duration to wait for completion of dynamic filters")
```
?

Reply:
Modified